src/confluent_kafka/src/Admin.c

```diff
@@ -2835,6 +3199,121 @@ static PyObject *Admin_c_DescribeConsumerGroupsResults_to_py(
+
+        scram_credential_info_cnt = rd_kafka_UserScramCredentialsDescription_scramcredentialinfo_count(description);
+        scram_credential_infos = PyList_New(scram_credential_info_cnt);
+        for(i=0; i < scram_credential_info_cnt; i++){
+                const rd_kafka_ScramCredentialInfo_t *scram_credential_info =
+                        rd_kafka_UserScramCredentialsDescription_scramcredentialinfo(description,i);
+                PyList_SET_ITEM(scram_credential_infos,i,
+                        Admin_c_ScramCredentialInfo_to_py(
+                                scram_credential_info));
+        }
+
+        PyDict_SetItemString(kwargs,"scram_credential_infos",scram_credential_infos);
+        args = PyTuple_New(0);
+        UserScramCredentialsDescription_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                                   "UserScramCredentialsDescription");
+        result = PyObject_Call(UserScramCredentialsDescription_type, args, kwargs);
+        Py_DECREF(args);
+        Py_DECREF(kwargs);
+        Py_DECREF(UserScramCredentialsDescription_type);
+        return result;
+}
+
+static PyObject *Admin_c_DescribeUserScramCredentialsResult_to_py(const rd_kafka_DescribeUserScramCredentials_result_t *result_event) {
+        PyObject *result = NULL;
+        const rd_kafka_UserScramCredentialsDescription_t **descriptions;
+        size_t description_cnt;
+        size_t i;
+
+        result = PyDict_New();
+        descriptions = rd_kafka_DescribeUserScramCredentials_result_descriptions(result_event, &description_cnt);
+        for(i=0; i<description_cnt; i++){
+                const char *username;
+                const rd_kafka_error_t *error;
+                rd_kafka_resp_err_t err;
+                const rd_kafka_UserScramCredentialsDescription_t *description = descriptions[i];
+                username = rd_kafka_UserScramCredentialsDescription_user(description);
+                error = rd_kafka_UserScramCredentialsDescription_error(description);
+                err = rd_kafka_error_code(error);
+                if (err) {
+                        PyDict_SetItemString(result, username,
+                                KafkaError_new_or_None(err,
+                                        rd_kafka_error_string(error)));
+                } else {
+                        PyDict_SetItemString(result, username,
+                                Admin_c_UserScramCredentialsDescription_to_py(description));
+                }
+        }
+        return result;
+}
+
+static PyObject *Admin_c_AlterUserScramCredentialsResult_to_py(const rd_kafka_AlterUserScramCredentials_result_t *result_event) {
+        PyObject *result = NULL;
+        size_t cnt, i;
+        const rd_kafka_AlterUserScramCredentials_result_response_t **responses =
+                rd_kafka_AlterUserScramCredentials_result_responses(result_event, &cnt);
```

Comment:
Let's move this to the below switch case to make it consistent with other functions. Also, I think its better to convert `c_responses` than to convert `result_event`.